corehq/apps/app_manager/models.py

```diff
@@ -4468,10 +4468,32 @@ def make_build(self, comment=None, user_id=None):
         assert copy._id
         prune_auto_generated_builds.delay(self.domain, self._id)
 
-        self.check_build_dependencies(new_build=copy)
+        self.analyse_app_build(new_build=copy)
 
         return copy
 
+    def analyse_app_build(self, new_build):
```

Comment:
+1 on doing this async.
Gathering metrics for our use should not delay user interaction, seems unreasonable.